scripts/discovery/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# update CA certificates
+update-ca-certificates 2>/dev/null
```

Comment:
Can we do this at Docker image build time?

Reply:
no, we need user provided CAs in ConfigMap mounted in `/usr/local/share/ca-certificates`